Content.Shared/_CM14/Xenos/Construction/Nest/XenoNestSystem.cs

```diff
@@ -169,7 +169,7 @@ private void OnNestSurfaceDoAfter(Entity<XenoNestSurfaceComponent> ent, ref Xeno
 
         // TODO CM14 make a method to do this
         var victimName = Identity.Name(victim, EntityManager, args.User);
-        _popup.PopupClient($"You secrete a thick, vile resin, securing {victimName} into the alien nest!", args.User, args.User);
+        _popup.PopupClient(Loc.GetString("cm-xeno-nest-securing-self", ("target", victimName)), args.User, args.User);
```

Comment:
Don't need the line above anymore.
```suggestion
        _popup.PopupClient(Loc.GetString("cm-xeno-nest-securing-self", ("target", victim)), args.User, args.User);
```